internal/versions/versions.go

```diff
@@ -62,14 +62,14 @@ const (
 	// JoinImage image of Constellation join service.
 	JoinImage = "ghcr.io/edgelesssys/constellation/join-service:v2.6.0-pre.0.20230127131021-e174146e0c93@sha256:6aeaab979ed0297fcab658d77b51ee26d9f1ce380d0add30f4d14648f34bd033" // renovate:container
 	// KeyServiceImage image of Constellation KMS server.
-	KeyServiceImage = "ghcr.io/edgelesssys/constellation/key-service:v2.6.0-pre.0.20230131161703-e0354826e058@sha256:6b25f9519f9a2ff40d08f4d74cdab8b66c4ad8c27ec64e0f7f5e460ac76b6b63" // renovate:container
+	KeyServiceImage = "ghcr.io/edgelesssys/constellation/key-service:v2.6.0-pre.0.20230206104741-826ebebbd004@sha256:e82624bc9889497589672f56680a9b5e4f8f6ead5673d7fcc6507ff534218b3d" // renovate:container
```

Comment:
This commit isn't on main branch, and the image wasn't tagged with latest. See https://github.com/edgelesssys/constellation/pkgs/container/constellation%2Fkey-service/67730075?tag=v2.6.0-pre.0.20230206104741-826ebebbd004
This shouldn't happen and is likely a bug in our renovate setup/config.

Reply:
The bug is likely not in the workflow, but for reference, the run that built that image: https://github.com/edgelesssys/constellation/actions/runs/4102839946/jobs/7076294098

Reply:
The commit the image was built on: https://github.com/edgelesssys/constellation/commit/826ebebbd004